crates/app/src/lib.rs

```diff
@@ -20,15 +20,18 @@ pub use trees::*;
 use std::fmt::Display;
 use std::sync::Arc;
 
-use drawbridge_store::{Get, Memory};
+use drawbridge_store::{Get, Storage};
 use drawbridge_type::{RepositoryName, TagName, TreePath};
 
 use axum::http::StatusCode;
 use tokio::sync::RwLock;
 
-pub type RepoStore = RwLock<Memory<RepositoryName>>;
-pub type TagStore = RwLock<Memory<(RepositoryName, TagName)>>;
-pub type TreeStore = RwLock<Memory<(RepositoryName, TagName, TreePath)>>;
+pub type RepoStoreKey = RepositoryName;
```

Comment:
Please try to avoid type aliases as a general rule. The only reason they were there was to avoid overly long type names and a corresponding clippy warning as a temporary workaround. I should have left a comment about that.